components/HeaderBar.vue

```diff
@@ -50,7 +49,20 @@ export default Vue.extend({
   },
   methods: {
     rubles,
-    openSearcher: toggleSearcher,
+    toggleSearcher () {
```

Comment:
Great Job Roman..
Better to use instance property From Vue js without local storage:
https://vuejs.org/v2/cookbook/adding-instance-properties.html
For example 
```suggestion
      data () {
    return {
      amount: 6234234,
      IsSearchBoxOpened: false,
    };
  },
  methods: {
    rubles,
    toggleSearcher () {
      const searchModalWindow = document.getElementById('searcher');
      if (searchModalWindow) {
        if (this.IsSearchBoxOpened) {
          searchModalWindow.style.display = 'none';
          this.IsSearchBoxOpened = false;
        } else {
          searchModalWindow.style.display = 'block';
          this.IsSearchBoxOpened = true;
        }
      }
    },
  },
```
![image](https://user-images.githubusercontent.com/14800560/84394919-38be5380-ac06-11ea-921e-6c267a67598c.png)
Also, see https://vuejs.org/v2/guide/conditional.html maybe you will change your mind of using the display property

Reply:
Read this interesting article about local storage https://dev.to/rdegges/please-stop-using-local-storage-1i04